File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.java
class Solution {
    public List<List<Integer>> findDifference(int[] nums1, int[] nums2) {
        List<Integer>l1 = new ArrayList<>();
        List<Integer>l2 = new ArrayList<>();
        for(int i : nums1){
            if(!contains(nums2,i) && !l1.contains(i)){
                l1.add(i);
            }
        }
        for(int j : nums2){
            if(!contains(nums1,j) && !l2.contains(j)){
                l2.add(j);
            }
        }
        List<List<Integer>> res = new ArrayList<>();
        res.add(l1);
        res.add(l2);
        return res;
    }
    public static boolean contains(int[] arr,int x){
        for(int i : arr){
            if(i == x){
                return true;
            }
        }
        return false;
    }
}

File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.py
class Solution(object):
    def findDifference(self, nums1, nums2):
        l1,l2 = [],[]
        for i in nums1:
            if i not in nums2 and i not in l1:
                l1.append(i)
        for j in nums2:
            if j not in nums1 and j not in l2:
                l2.append(j)
        return [l1,l2]
        